src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java

```diff
@@ -87,17 +87,16 @@
 import uk.gov.justice.hmpps.prison.service.OffenderLocationService;
 import uk.gov.justice.hmpps.prison.service.OffenderTransactionHistoryService;
 import uk.gov.justice.hmpps.prison.service.PrisonerReleaseAndTransferService;
-import uk.gov.justice.hmpps.prison.service.receiveandtransfer.BookingIntoPrisonService;
```

Comment:
Planning to add some release prisoner functionality to this package so renamed using the HMPPS terminology.